packages/xxd_standalone.rb

```diff
@@ -3,24 +3,24 @@
 class Xxd_standalone < Package
   description 'Hexdump utility from vim'
   homepage 'http://www.vim.org'
-  @_ver = '8.2.2725'
+  @_ver = '8.2.2771'
```

Comment:
Can this be updated to 8.2.2783?

Reply:
Yes, but the source file hasn't changed. It compiled to literally the same binary.